composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "pugx/multi-user-bundle",
+    "name": "fcpauldiaz/multi-user-bundle",
```

Comment:
No

Reply:
Sorry, didn't wanted to make a pull request for this repo. I wanted to update my fork from this repo.